source/_integrations/velux.markdown

```diff
@@ -67,6 +67,31 @@ automation:
     - service: velux.reboot_gateway
 ```
 
+## Attributes
+
+Cover entity with integrated rain sensor has extra attributes to represent the state of the limitation sensor.
+
+| Name | Description |
+| ---- | ----------- |
+| `limitation_min` | Min limitation of window. `limitation_min` => 0 no rain; `limitation_min` => 93 the window can only open for 7 % , so it means, its raining
+| `limitation_max` | Max limitation of window.
```

Comment:
When is this set?

Reply:
Sorry I am not completely clear what you mean?

The values is updated every 2 Minutes https://github.com/home-assistant/core/pull/64628/files#diff-122894d3c6501f90d7be2e080a84f0823a54b52ae37908983e1087390a87791fR36

Reply:
I mean what's the meaning of `limitation_max` and how can it be used? For `limitation_min` there's a clear explanation.